app/controllers/api/functions.php

```diff
@@ -117,6 +117,28 @@
         'activate' => true,
     ]));
 
+    $buildId = ID::unique();
+    $build = $dbForProject->createDocument('builds', new Document([
+        '$id' => $buildId,
+        '$permissions' => [],
+        'startTime' => null,
+        'deploymentInternalId' => $deployment->getInternalId(),
+        'deploymentId' => $deployment->getId(),
+        'status' => 'waiting',
```

Comment:
we have to stay consistent with the available build types her https://github.com/appwrite/appwrite/blob/feat-cancel-build-deployment/src/Appwrite/Utopia/Response/Model/Build.php#L26-L29

Waiting doesn't seem to one we use..

Reply:
Created a separate task to use enums for states